src/Routes/Routes.tsx

```diff
@@ -87,20 +81,20 @@ export const Routes = () => {
     [appRoutes, hasWallet, isUnstableRoute, location],
   )
 
+  const switchLocation = useMemo(() => location.state?.background || location, [location])
```

Comment:
Would probably rename this as the vernacular implies this switches location but meh, not really concerned as just looking at usage a few lines below would explain what this does

Reply:
agreed and fixed :)